ampersand-form-view.js

```diff
@@ -161,9 +196,22 @@ assign(FormView.prototype, Events, {
         fieldView.parent = this;
         fieldView.render();
         this.fieldContainerEl.appendChild(fieldView.el);
-    }
-});
+    },
+
+    getValue: function(name) {
+        var field = this.getField(name, true);
+        return field.value;
+    },
+
+    setValue: function(name, value) {
+        var field = this.getField(name, true);
+        field.setValue(value);
+        return this;
+    },
 
-FormView.extend = classExtend;
+    // deprecated
+    getData: function() {
```

Comment:
If this does become deprecated, perhaps a `console.warn` –with a deprecated warning message– might be appropriate here?